### PR TITLE
GANG WEED

### PR DESCRIPTION
**Pull request by boodaliboo**
gangsters can now buy a joint in there gang tool
#### Changelog

:cl:  
rscadd: Added weed to gang tool
/:cl:

### CODE REVIEW

**Comment by Moja10** [comment]
Perhaps the best bood PR ever made?

File: yogstation/code/game/gamemodes/gangs/gang_items.dm [hunk 1]
```diff
@@ -266,6 +266,13 @@
 	item_path = /obj/item/sharpener
```
Comment (redmoogle) [inline yogstation/code/game/gamemodes/gangs/gang_items.dm:267]: Remove the two extra spaces and make Gangweed not start with a cap

**Review by redmoogle — CHANGES_REQUESTED** [review]
Remove the two extra spaces and make Gangweed not start with a cap

**Comment by jcatjmeow** [comment]
since you didn't explain yourself I assume this is a meme pr

**Pull request closed without merging**